sopel/plugin.py

```diff
@@ -928,6 +932,12 @@ def handle_example_bugs(bot, trigger, match):
 
         More than one pattern can be provided as positional argument at once.
 
+    .. versionchanged:: 7.1
+
+        The ``trigger`` parameter now represents the same match as the
+        ``match`` parameter, making ``match`` an obsolete parameter, kept
+        for backward compatibility.
```

Comment:
Question is how long we should keep `match` around in the calls. Since it (_should be_) defined as a kwarg in plugin code (`match=None`), not passing it shouldn't cause fatal errors. Might make the code fail to work as expected, though, if it's written to ignore `trigger` and use `match` only.

Can we say it'll be removed in 9.0? That should be long enough.

Reply:
Yes. Note that I've seen `match` argument as named argument instead of keyword argument: nothing in the doc specify that anyway.